dev/com.ibm.ws.security.authentication.builtin/resources/OSGI-INF/l10n/metatype.properties

```
@@ -33,6 +33,10 @@ fillInGroupInformationForHashtableLogin.desc=Specifies whether the group informa
 useDisplayNameForSecurityName=Use displayName as securityName in customUR
 useDisplayNameForSecurityName.desc=Specifies whether to return displayName for programmatic APIs like getRemoteUser, getCallerPrincipal, getUserPrincipal when using Custom User Registry.
 
+ignoreCustomCacheKey=Ignore custom cache key in LtpaToken2 cookie
```

```suggestion
ignoreCustomCacheKey=Ignore the custom cache key in LtpaToken2 cookie.
```

---

This is a label.  It shouldn't have a period at the end of it.
```suggestion
ignoreCustomCacheKey=Ignore the custom cache key in LtpaToken2 cookie
```
or
```suggestion
ignoreCustomCacheKey=Ignore custom cache key in the LtpaToken2 cookie
```
Not sure which one is better.

---

Stick to the older one - Ignore custom cache key in LtpaToken2 cookie